app/qml/map/MapWrapper.qml

```diff
@@ -771,7 +779,17 @@ Item {
           {
             return qsTr( "Unknown accuracy" )
           }
-          return __inputUtils.formatNumber( __positionKit.horizontalAccuracy, accuracyButton.accuracyPrecision ) + " m"
+
+          let accuracyText = __inputUtils.formatNumber( __positionKit.horizontalAccuracy, accuracyButton.accuracyPrecision ) + " m"
+          if ( __appSettings.gpsAntennaHeight > 0 )
+          {
+            let gpsText = __inputUtils.formatNumber( __appSettings.gpsAntennaHeight, 3 ) + " m"
```

Comment:
```suggestion
            let gpsText = Number( __appSettings.gpsAntennaHeight.toFixed( 3 ) ) + " m"
```

This will help you remove the trailing zeros <img width="182" alt="image" src="https://user-images.githubusercontent.com/22449698/216956067-247dbd14-49cb-482c-8420-274d2792abc1.png">